packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Event Hooks
+excerpt: Use Okta events to drive custom process flows.
+---
+
+# Event Hooks
```

Comment:
A few more things we should add in general - 
1. Rate limits. Currently, up to a maximum of 100k events are delivered in a 24hr period.
2. Limit on number of event hooks that can be created (10).
3. Okta currently does not provide any SLA for time between event publication and event delivery. @ankurdatta-okta can clarify more here

Reply:
Adding things so I don't forget - 
4. Best Practices? (especially around not waiting for workflow to complete before returning response to Okta)

Reply:
Added:
- New section "Rate Limits", saying "Event hooks are limited to sending 100 000 events per 24-hour period."
- Addition to section "What are Okta Event Hooks?", saying: "You can create a maximum of ten event hooks in your org. Each event hook can be configured to deliver multiple event types."
- Addition to "Ongoing Event Delivery" section, saying: "No guarantee of maximum delay between event occurrence and delivery is currently specified."
 - Addition to "Your Service's Responses to Event Delivery Requests" section, saying: "As a best practice, you should return the HTTP response immediately, rather than waiting for any of your own internal process flows triggered by the event to complete."

(All changes are in new pull request: https://github.com/okta/okta-developer-docs/pull/227)